funcx_endpoint/tests/test_rabbit_mq/test_endpoint.py

```diff
@@ -0,0 +1,124 @@
+import json
+import logging
+import multiprocessing
+import random
+import uuid
+
+import pika
+
+from funcx_endpoint.endpoint.rabbit_mq import (
+    ResultQueuePublisher,
+    ResultQueueSubscriber,
+    TaskQueuePublisher,
+    TaskQueueSubscriber,
+)
+
+LOG_FORMAT = "%(levelname) -10s %(asctime)s %(name) -20s %(lineno) -5d: %(message)s"
+logging.basicConfig(level=logging.INFO, format=LOG_FORMAT)
```

Comment:
`pytest` has automatic log capturing, so I don't think there's any need to do `basicConfig` or set a format in tests. You can just initialize and use loggers throughout.

Like the capsys fixture, there's a caplog fixture you can use to tune it if absolutely necessary:
https://docs.pytest.org/en/7.0.x/how-to/logging.html